src/core/Wizard/Wizard.tsx

```diff
@@ -96,6 +103,7 @@ export const Wizard = ({
               totalSteps={steps.length}
               type={type}
               onClick={onStepClick}
+              description={s.description}
```

Comment:
I'm wondering if we should use `name` when `description` is not available. Will be useful when name is not visible ("Long")

Reply:
@veekeys @bentleyvk Thoughts?

Reply:
And what would be the API if user does not want to have a tooltip?

Reply:
I guess user could pass `''` idk.

Maybe no need